projects/vanilla/src/styles/preview.scss

```diff
@@ -49,13 +49,13 @@ $highlights: map-merge(map-merge($extracts-highlights, $terms-highlights), $meta
 
 @mixin highlight($term) {
```

Comment:
The changes in this file seem unrelated to the other. Did you commit them by mistake?

Reply:
You right... I will remove it